oidc-js/src/utils/sign-in.ts

```diff
@@ -66,7 +67,11 @@ import {
     isWebWorkerConfig
 } from "../models";
 import { AuthenticatedUserInterface } from "../models/authenticated-user";
-import { TokenRequestHeader, TokenResponseInterface } from "../models/token-response";
+import {
+    DecodedIdTokenPayloadInterface,
+    TokenRequestHeader,
+    TokenResponseInterface
+} from "../models/token-response";
```

Comment:
Shall we export `token-response` from `models` index so that we can reuse the above `models` import.

Reply:
Resolved by be00224b